src/main/java/net/fabricmc/loader/metadata/ModMetadataV1.java

```diff
@@ -340,7 +340,7 @@ public Person deserialize(JsonElement json, Type typeOfT, JsonDeserializationCon
 					person.name = obj.get("name").getAsString();
 					if (obj.has("contact")) {
 						person.contact = new MapBackedContactInformation(
-							context.deserialize(obj.get("contact"), TypeToken.getParameterized(HashMap.class, String.class, String.class).getType())
+							context.deserialize(obj.get("contact"), new TypeToken<HashMap<String, String>>(){}.getType())
```

Comment:
Why is this needed

Reply:
legacy guava support

Reply:
Ideally we would work around this by detaching loader from guava so you don't need to worry about that.

Reply:
Hmm, is this guava or gson type token? Either way we should be shading both libraries or moving away now.